platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/module/telemetry/PerformanceEvent.java

```diff
@@ -1,116 +1,66 @@
 package com.mapbox.mapboxsdk.module.telemetry;
 
-import com.google.gson.Gson;
+import android.annotation.SuppressLint;
+import android.os.Bundle;
 
+import com.google.gson.Gson;
 import com.google.gson.JsonObject;
-
 import com.google.gson.reflect.TypeToken;
-import com.mapbox.android.telemetry.Event;
 
-import android.os.Bundle;
-import android.os.Parcel;
-
-
-import java.text.SimpleDateFormat;
 import java.util.ArrayList;
-import java.util.Date;
 import java.util.List;
-import java.util.Locale;
 
 /**
  * Generic Performance Event that can be used for performance measurements.
  * Customer measurements can be added to the bundle.
- *
+ * <p>
  * Bundle is expected to have following properties:
  * "attributes", "counters", and "metadata" with String values.
- *
+ * <p>
  * Attributes: a string representing an array of name/string value pair objects.
  * Counters: a string representing an array of name/number value pair objects.
  * Metadata is a string representation of a JsonObject with string values.
- *
- *  Here is an example of a Performance event bundle data:
- *
+ * <p>
+ * Here is an example of a Performance event bundle data:
+ * <p>
  * "attributes": [{ "name": "style_id", "value": "mapbox://styles/mapbox/streets-v10"}]
- *
+ * <p>
```

Comment:
having `<p>` is :+1: for getter better readable output but the javadoc tool will warn that the `<p>` aren't enclosed, can you add `</p>`?

Reply:
OK